corehq/apps/reports/tests/test_scheduled_reports.py

```diff
@@ -158,6 +160,39 @@ def testMonthlyReportOnTheEndOfTheMonthDaysAfter31DontCount(self):
         self._check('monthly', datetime(2014, 10, 31, 12, 0), 1)
 
 
+class ScheduledReportSendingTest(TestCase):
+    dependent_apps = [
+        'django_digest', 'auditcare', 'django_prbac',
+        'corehq.apps.accounting',
+        'corehq.apps.domain',
+        'corehq.apps.users',
+        'corehq.apps.tzmigration',
+        'corehq.apps.userreports',
+    ]
```

Comment:
this doesn't do anything until https://github.com/dimagi/commcare-hq/pull/9220/ is merged, but i used it locally to make sniffing a lot nicer